src/verify/PhoneVerificationStartScreen.tsx

```diff
@@ -0,0 +1,293 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { initializeAccount, setPhoneNumber } from 'src/account/actions'
+import { defaultCountryCodeSelector, e164NumberSelector } from 'src/account/selectors'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { registrationStepsSelector } from 'src/app/selectors'
+import BackButton from 'src/components/BackButton'
+import Button, { BtnTypes } from 'src/components/Button'
+import Dialog from 'src/components/Dialog'
+import KeyboardAwareScrollView from 'src/components/KeyboardAwareScrollView'
+import PhoneNumberInput from 'src/components/PhoneNumberInput'
+import TextButton from 'src/components/TextButton'
+import i18n from 'src/i18n'
+import { setHasSeenVerificationNux } from 'src/identity/actions'
+import { HeaderTitleWithSubtitle } from 'src/navigator/Headers'
+import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarTextButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import { waitUntilSagasFinishLoading } from 'src/redux/sagas'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { getCountryFeatures } from 'src/utils/countryFeatures'
+import { getPhoneNumberState } from 'src/verify/utils'
+import { walletAddressSelector } from 'src/web3/selectors'
+
+function PhoneVerificationStartScreen({
+  route,
+  navigation,
+}: StackScreenProps<StackParamList, Screens.VerificationEducationScreen>) {
+  const [showLearnMoreDialog, setShowLearnMoreDialog] = useState(false)
+  const [showSkipDialog, setShowSkipDialog] = useState(false)
+  const [phoneNumberInfo, setPhoneNumberInfo] = useState(() =>
+    getPhoneNumberState(
+      cachedNumber || '',
+      cachedCountryCallingCode || '',
+      route.params?.selectedCountryCodeAlpha2 || RNLocalize.getCountry()
+    )
+  )
+
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  const headerHeight = useHeaderHeight()
+
+  const account = useSelector(walletAddressSelector)
+  const cachedNumber = useSelector(e164NumberSelector)
+  const cachedCountryCallingCode = useSelector(defaultCountryCodeSelector)
+  const walletAddress = useSelector(walletAddressSelector)
+  const { step, totalSteps } = useSelector(registrationStepsSelector)
+
+  const countries = useMemo(() => new Countries(i18n.language), [i18n.language])
+  const country = phoneNumberInfo.countryCodeAlpha2
+    ? countries.getCountryByCodeAlpha2(phoneNumberInfo.countryCodeAlpha2)
+    : undefined
+
+  const onPressStart = async () => {
+    if (!canUsePhoneNumber()) {
+      return
+    }
+    // TODO figure out what this nux thing does
+    dispatch(setHasSeenVerificationNux(true))
+    // TODO dispatch action to call start verification service
+    navigate(Screens.VerificationInputScreen, {
```

Comment:
Could we directly navigate to the new `PhoneVerificationInputScreen` instead?
This would avoid touching the existing `VerificationInputScreen` completely.
I think only the decentralized `VerificationLoadingScreen` navigates to it. So if we've already branches the `VerificationEducationScreen`, we're fine.

Reply:
ah, do you mean something like this? https://github.com/valora-inc/wallet/pull/2894/commits/6f0bf233ade2559d15ac7903744848a740175225 i was in two minds about what is best to do here, i initially had a new PhoneVerificationInputScreen in the navigator but i changed it to do the switch inside `VerificationInputScreen` so that we can keep the namespace when removing the decentralised flow later on. :/ but indeed it does touch the existing screen. I'll think of a better screen name and put it back in the navigator

Reply:
yes, ah well either way is fine then.

Reply:
i renamed the PhoneVerificationInputScreen to VerificationCodeInputScreen - it's a bit confusing to have `VerificationCodeInputScreen` and `VerificationInputScreen` but when we remove the decentralised flow we'll be left with `VerificationEducationScreen` and `VerificationCodeInputScreen` which is not so bad :)